mcp/src/graph/neo4j.ts

```diff
@@ -512,6 +530,143 @@ class Db {
     }
   }
 
+  async create_hint(question: string, answer: string, embeddings: number[]) {
+    const session = this.driver.session();
+    const name = question.slice(0, 80);
+    const node_key = `Hint|:|${name}|:|hint://generated|:|0`;
```

Comment:
can you clean up the node key? i think we already have function to clean string (no special characters, strings, etc)